4-fleetscope/modules/env_baseline/policy.tf

```diff
@@ -27,11 +27,45 @@ resource "google_gke_hub_feature" "feature" {
           bundles {
             bundle = "pss-baseline-v2022"
           }
+          bundles {
+            bundle = "policy-essentials-v2022"
+          }
           template_library {
             installation = "ALL"
           }
         }
+        referential_rules_enabled = true
+      }
+    }
+  }
+}
+
+resource "google_gke_hub_feature_membership" "poco_feature_member" {
+  for_each = toset(var.cluster_membership_ids)
+  location = "global"
+  project  = var.cluster_project_id
```

Comment:
fleet_project_id ?

Reply:
Yes, good catch, I started this particular PR prior to those being swung over.